flow/project.py

```diff
@@ -3512,7 +3527,7 @@ def _is_selected_aggregate(aggregate, jobs):
     def _get_aggregate_from_id(self, id):
         # Iterate over all the instances of stored aggregates and search for the
         # aggregate in those instances.
```

Comment:
Is this just iterating over groups of aggregate ids? is 
```suggestion
        # Iterate over all the instances of stored aggregate ids and search for the
        # aggregate id in those instances.
```
more accurate, or am I misunderstanding?

Reply:
Your comment suggestion is partially correct. The aggregate stores are mappings from `aggregate_id` to `aggregate`. I suggest we change the second word (we search for an aggregate _id_) but not the first (the instances are "stored aggregates", not just ids).
```suggestion
        # Iterate over all the instances of stored aggregates and search for the
        # aggregate id in those instances.
```